packages/babel-parser/test/fixtures/estree/export/decorator-before-export/output.json

```diff
@@ -0,0 +1,70 @@
+{
+  "type": "File",
+  "start":0,"end":51,"loc":{"start":{"line":1,"column":0},"end":{"line":2,"column":28}},
+  "program": {
+    "type": "Program",
+    "start":0,"end":51,"loc":{"start":{"line":1,"column":0},"end":{"line":2,"column":28}},
+    "sourceType": "module",
+    "interpreter": null,
+    "body": [
+      {
+        "type": "ExportNamedDeclaration",
+        "start":5,"end":22,"loc":{"start":{"line":1,"column":5},"end":{"line":1,"column":22}},
```

Comment:
When `estree` is enabled, we restore the pre-7.20 behaviour: the `ExportNamedDeclaration` now starts at the `export` token.